GoMeet/src/main/java/com/google/sps/data/ServletUtil.java

```diff
@@ -72,16 +72,17 @@ public static void sendErrorResponse(HttpServletResponse response, int status, S
    * @return the decoded URI
    */
   public static String decodeUri(String encodedUri) {
+    String result; 
     // Decode "%2C" to ","
-    encodedUri.replace("%2C", ",");
```

Comment:
Consider using [URLDecoder](https://docs.oracle.com/javase/7/docs/api/java/net/URLDecoder.html) to do this instead.

Reply:
+1, it's generally better to use preexisting tools, but now that you've already made your own iteration, it's not necessary.

Reply:
Ok I'll keep this in mind for next time